Client/qtTeamTalk/changestatusdlg.cpp

```diff
@@ -61,10 +61,12 @@ void ChangeStatusDlg::slotAccepted()
         m_user.nStatusMode |= STATUSMODE_AVAILABLE;
     else if(ui.awayBtn->isChecked())
         m_user.nStatusMode |= STATUSMODE_AWAY;
+                    TT_DoChangeStatus(ttInst, m_statusmode, _W(tr("Away")));

```

Comment:
The status message should be what the user has specified in this case. E.g. away status set to "I'll be back in 10 minutes".

Reply:
I've tried to implemente this in 99a1867, I really don't sûre of result :)